bin/fuel-core/Cargo.toml

```diff
@@ -47,8 +49,19 @@ test-case = { workspace = true }
 default = ["env", "relayer", "rocksdb"]
 env = ["dep:dotenvy"]
 p2p = ["fuel-core/p2p", "const_format"]
-relayer = ["fuel-core/relayer", "dep:url", "dep:serde_json"]
-rocksdb = ["fuel-core/rocksdb"]
-rocksdb-production = ["fuel-core/rocksdb-production"]
+relayer = ["fuel-core/relayer", "dep:url"]
+parquet = ["fuel-core-chain-config/parquet"]
+rocksdb = [
```

Comment:
BTW, if you want, you can create a separate PR on the `master` to use your formatting for the toml files. But you also need to add a CI action that will verify that formating is the same for everyone=)